src/importer.rs

```diff
@@ -159,7 +161,18 @@ impl<C: ObjCache> Importer<C> {
                 let models = match self.cache.get_by_checksum(checksum) {
                     Some(models) => models.clone(),
                     None => {
-                        let (tobj_models, _) = obj_buf_into_tobj(&mut file_contents.as_slice())?;
+                        let (mut tobj_models, _) =
+                            obj_buf_into_tobj(&mut file_contents.as_slice())?;
+                        let mut index = 0;
+
+                        while index != tobj_models.len() {
+                            if tobj_models[index].mesh.positions.is_empty() {
+                                tobj_models.remove(index);
+                            } else {
+                                index += 1;
+                            }
+                        }
```

Comment:
Could we move the functionality you added to `tobj_to_internal`? Empty OBJs are completely valid. It is just our software doesn't accept them :joy: